fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1066,12 +1083,9 @@ public void handleInsertOverwrite(InsertStmt insertStmt) {
 
     public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
         if (stmt.isExplain()) {
-            handleExplainStmt(execPlan.getExplainString(stmt.getExplainLevel()));
+            handleExplainStmt(buildExplainString(execPlan));
             return;
         }
-        if (context.getQueryDetail() != null) {
-            context.getQueryDetail().setExplain(execPlan.getExplainString(TExplainLevel.NORMAL));
-        }
```

Comment:
Is it possible that `stmt.isExplain==false` and `context.getQueryDetail() != null`?

Reply:
fixed